exercises/practice/anagram/test/anagram_test.dart

```diff
@@ -1,31 +1,21 @@
-import 'package:anagram/anagram.dart';
 import 'package:test/test.dart';
+import 'package:anagram/anagram.dart';
 
 final anagram = Anagram();
 
 void main() {
-  group("simple tests", simpleTests);
-
-  group("slightly less simple tests", slightlyLessSimpleTests);
-
-  group("more challenging tests", moreChallengingTests);
-
-  /// Edge cases are scenarios that are problems that occur under specific conditions
-  /// Read more: https://en.wikipedia.org/wiki/Edge_case
-  group("edge case tests", edgeCaseTests);
+  group("Anagram: anagram tests - ", anagramTests);
```

Comment:
A top-level group is enough for this exercise, as canonical-data does not have any grouping. We do not need to extract out test cases to a named function `anagramTests`, a closure would suffice here. But I can understand that this would be special casing in your test generator.

Reply:
No, that's no problem, and a good shout. I'll tweak it so that it does it that way.